public/docs/ts/latest/guide/i18n.jade

```diff
@@ -86,12 +86,12 @@ a(id="angular2-i18n")
   Whatever approach we decide to use, we must update our templates to define which strings will be translated.
  
   
-a(id="i18n-directive")
+a(id="i18n-directives")
 .l-main-section
 :marked
-  ### i18n directive
+  ### i18n directives
```

Comment:
Also, does this `i18n` attribute have any meaning outside of Angular? e.g. is it a convention, or part of a spec.